@vates/types/package.json

```diff
@@ -37,5 +32,8 @@
   "author": {
     "name": "Vates SAS",
     "url": "https://vates.fr"
+  },
+  "engines": {
+    "node": ">=8.10"
```

Comment:
The script provides default values, feel free to update them if necessary.

We need to support at least the last 2 maintained LTS.

I will bump the script's default value because it does not make sense to support Node 8 :joy: